### PR TITLE
will use vhost-specific redirect if redirect is given

### PR DESCRIPTION
**Pull request by orthagonal**
will fallback to any matching non-vhost redirect otherwise

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -12,26 +14,42 @@ module.exports = function(server, options, next) {
...
-
-    server.log(['hapi-redirects', 'error'], { path: request.path });
+    server.log(['hapi-redirects', 'error'], { path: request.path, vhost : vhost });
     reply(new Error('invalid redirect'));
   };
 
-  for(var route in options.redirects) {
-    routes.push({
-      path: route,
+  function registerRoute(path, vhost){
```
Comment (jgallen23) [inline lib/index.js]: lets make sure we are consistent stylistically.  `var registerRoute = function()`

**Comment by jgallen23** [comment]
can you merge in master into this?

**Comment by orthagonal** [comment]
Will do.

**Comment by jgallen23** [comment]
it looks like you merged this into master...

**Comment by jgallen23** [comment]
its fine for now, we just need to make sure we don't merge until things have been reviewed

**Pull request merged**